src/test/integration/org/apache/cassandra/sidecar/testing/CassandraSidecarTestContext.java

```diff
@@ -87,8 +122,13 @@ private InstancesConfig buildInstancesConfig(CassandraVersionProvider versionPro
             // Use the parent of the first data directory as the staging directory
             Path dataDirParentPath = Paths.get(dataDirectories[0]).getParent();
             assertThat(dataDirParentPath).isNotNull()
-                                         .exists();
-            String uploadsStagingDirectory = dataDirParentPath.resolve("staging").toFile().getAbsolutePath();
+                      .exists();
+            Path stagingPath = dataDirParentPath.resolve("staging");
+            String uploadsStagingDirectory = stagingPath.toFile().getAbsolutePath();
+            if (!Files.exists(stagingPath))
+            {
+                Files.createDirectory(stagingPath);
+            }
```

Comment:
similar to `mkdir -p ...`
```suggestion
            Files.createDirectories(stagingPath);
```